partner_contact_gender/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2014-2015 Grupo ESOC <www.grupoesoc.es>
+# Copyright 2017-Apertoso N.V. (<http://www.apertoso.be>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Contact gender",
+    "summary": "Add gender field to contacts",
+    'version': '11.0.1.0.0',
+    "category": "Customer Relationship Management",
+    "website": "https://www.tecnativa.com/",
```

Comment:
@NL66278 Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)